client/src/main/kotlin/no/nav/su/se/bakover/client/person/AdGraphApiClient.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.su.se.bakover.client.person
+
+import arrow.core.Either
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.github.kittinunf.fuel.core.extensions.authentication
+import com.github.kittinunf.fuel.httpGet
+import no.nav.su.se.bakover.client.azure.OAuth
+import no.nav.su.se.bakover.common.objectMapper
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+data class AdGraphResponse(
+    val onPremisesSamAccountName: String,
```

Comment:
Er alle disse alltid fyllt ut for alle?

Reply:
Jeg tror de i praksis skal være det ja.
Fra doc:
> The property is only populated for customers who are synchronizing their on-premises directory to Azure Active Directory via Azure AD Connect.

Kanskje vi likevel skulle ha markert dem som nullable i tilfelle det i framtiden sluttes med synkroniseringen mot on-premise AD?